src/services/notifications/chatTypes.ts

```diff
@@ -40,15 +41,28 @@ export interface ISendMessage {
   chatRoomId: number;
   text: string;
 }
+interface INotificationContent {
+  id: number;
+  message: string;
+}
+export interface INotification {
+  id: number;
+  isRead: boolean;
+  type: NotificationTypeEnum;
+  text: string;
+  createdAt: Date;
+  roomId: number;
+  message: INotificationContent | null;
```

Comment:
`message?: INotificationContent` better make like that

Reply:
message?: INotificationContent === message: INotificationContent | undefined
in my case I can receive INotificationContent or null. more type safe.